documentation/deliverables/individual_deliverables/sprint_reporting_dashboard.md

```diff
@@ -0,0 +1,264 @@
+
+**Goals**
+
+- Ensure that we are collecting the data needed to calculate proposed metrics
+- Begin to create the infrastructure required for more advanced data analytics
+- Begin to establish a framework to measure project impact and success
+- Build public trust in our approach through transparency around core metrics
+- Improve our ability to understand and manage team capacity and delivery velocity
+
+### User Stories
+<!-- Required -->
+
+- As a **project maintainer**, I want:
+  - to be able to provide input on which metrics we publish and how they are presented in the dashboard, so that I can use the dashboard to meaningfully plan and manage delivery on the project.
+  - to be able to provide additional details about the metrics included in the dashboard, so that other stakeholders know how to interpret these metrics correctly in the context of the project.
+- As an **internal stakeholder**, I want:
+  - the dashboard's key insights to be easy to understand at a glance, so that I don't need to spend a lot of time learning how to interpret the metrics correctly.
+  - to be able to access the dashboard from the HHS network, so that I can easily monitor our metrics when I'm in the office.
+  - to understand how the metrics are calculated, so that I can explain them to other stakeholders.
+  - the data behind the dashboard to be updated regularly, so that I'm not sharing outdated information with other stakeholders.
+- As a **member of the public**, I want:
+  - to be able to easily navigate between the dashboard and other project resources (e.g. Simpler.Grants.gov or GitHub), so that I don't have to bookmark or remember the links for each resource separately.
+  - all of the key project dashboards to be accessible in a central location, so that I don't have bookmark or remember multiple links to view all of the metrics.
+- As an **open source contributor**, I want:
+  - to have access to the data behind the dashboard, so that I can explore and analyze the data myself.
+  - to access the source code behind the dashboard, so that I can use this dashboard for my own project or submit code to improve upon the existing dashboard.
+
+## Technical description
+
+### Dashboard
+
+This deliverable should implement the architecture to build and host dashboards that enables us to iteratively publish operational metrics related to the project.
+
+Some options to consider include:
+
+1. Analytics API + separate dashboard UI
+2. Custom dashboard application (e.g. Plotly Dash or R Shiny app)
+3. Open source dashboard solution (e.g. Metabase or Redash)
+4. Software as a Service (SaaS) dashboard solutions (e.g. PowerBI or Tableau)
```

Comment:
I'm not sure if this fits within your existing 4 options or is a 5th option, but we could call out just having Jupyter notebooks (or similar) that connect to the database over APIs or direct database connections. 

These are easy to build and less easy to look at, so there are pros and cons.

Reply:
Yeah the real evaluation of these options will be in this ADR #936 but it's good to highlight it as an option here. Updated it in this commit: [fix: Incorporates PR feedback](https://github.com/HHS/simpler-grants-gov/pull/516/commits/9652deea7dd042058dfea99b6c7a5be27738e49d)